hack/bin/deps.bzl

```diff
@@ -223,6 +224,39 @@ def install_kubectl():
         urls = ["https://storage.googleapis.com/kubernetes-release/release/v1.18.0/bin/linux/amd64/kubectl"],
     )
 
+# Define rules for different golangci-lint versions
+def install_golangci_lint():
+    http_archive(
+        name = "golangci_lint_1_42_darwin",
```

Comment:
Including the version here means that when we update this, we need to do it in two places (here and in BUILD.bazel). Can we omit the version (i.e. `golangci_lint_darwin`)?

Reply:
Yes, makes sense. Removed